### PR TITLE
increment version support table entry for 0.56.* to 0.56.2

### PR DESCRIPTION
**Pull request by esc**
With https://github.com/numba/numba/pull/8414/files the version support table on `main` must also be updated.

### CODE REVIEW

**Review by Sohlstyce — APPROVED** [review]
Fixing an overlook, should be fine.

File: docs/source/user/installing.rst [hunk 1]
```diff
@@ -278,7 +278,7 @@ information.
 +===========+==============+===========================+============================+==============================+===================+=============================+
 | 0.57.x    | TBC          | 3.8.x <= version < 3.12   | 1.19 <= version < 1.24     | 0.40.x                       | 11.x              | 2021.x                      |
 +-----------+--------------+---------------------------+----------------------------+------------------------------+-------------------+-----------------------------+
-| 0.56.1    | 2022-09-01   | 3.7.x <= version < 3.11   | 1.18 <= version < 1.24     | 0.39.x                       | 11.x              | 2021.x                      |
+| 0.56.2    | 2022-09-01   | 3.7.x <= version < 3.11   | 1.18 <= version < 1.24     | 0.39.x                       | 11.x              | 2021.x                      |
```
Comment (stuartarchibald) [inline docs/source/user/installing.rst:281]: Is it worth including a line about 0.56.1 with a note about it not being shipped?

Comment (esc) [reply docs/source/user/installing.rst:281]: maybe, yes, it does appear like that in the changelog

Comment (stuartarchibald) [reply docs/source/user/installing.rst:281]: Whilst perhaps not strictly necessary, given this table is the reference point for package maintainers to work out what works/is tested, I'm inclined to say include it, on the basis it'll save them having to go looking further to figure out where 0.56.1 went. Thanks!

**Review by stuartarchibald — DISMISSED** [review]
Thanks for the patch.

**Comment by gmarkall** [comment]
Marked as abandoned following triage discussion - new PR to follow

**Pull request closed without merging**